security-council-rehearsals/templates/r1-hello-council/justfile

```diff
@@ -1,9 +1,9 @@
 set dotenv-load
+export location := `pwd`
+export OWNER_SAFE := env_var("COUNCIL_SAFE")
```

Comment:
This change wasn't strictly necessary, but it makes this file more consistent with the r2 and r4 rehearsal justfiles.